prac_06/guitars.py

```diff
@@ -1,12 +1,55 @@
 """Guitar exercise - Program using Guitar class
-Time expected:  min
+Time expected: 30 min
        actual:  min
 """
 
 
 def main():
-    """ """
-    pass
+    """Ask user for guitar data and then present information
+    in a nicely formatted table."""
+    print("My guitars!")
+
+    name = get_non_blank_string("Name: ")
+    year = get_positive_integer("Year: ")
+    cost = get_positive_float("Cost: $")
+
+
+def get_non_blank_string(prompt):
+    """Gets a non-blank string from the user with prompt message.
+    Continues to ask until valid input is given."""
+    string = input(prompt).strip()
+    while string == "":
+        print("Cannot be blank")
+        string = input(prompt).strip()
+    return string
+
+
+def get_positive_integer(prompt):
```

Comment:
This is good but I don't think it was asked for.

Reply:
True, we didn't need input error checking for this exercise. I just copied these from a previous exercise 😄